### DISCUSSION TITLE
How to block Yandex image ads searching?

### DISCUSSION THREAD

**Discussion by Jokopentil** [uBlockOrigin/uBOL-home#81] [Q&A]
What filter turn on

**Comment by MasterKia** [comment]
Install uBlock Origin and add this filter list:
https://easylist-downloads.adblockplus.org/cntblock.txt

**Comment by Jokopentil** [comment]
I use chrome ads can block but ads bicome blank white

Pada Kam, 26 Okt 2023 11.20, MasterKia ***@***.***> menulis:

**Reply by MasterKia** [reply]
> add this filter list:
https://easylist-downloads.adblockplus.org/cntblock.txt
